spirv_cross.cpp

```diff
@@ -96,10 +96,8 @@ bool Compiler::variable_storage_is_aliased(const SPIRVariable &v)
 bool Compiler::block_is_pure(const SPIRBlock &block)
 {
 	// This is a global side effect of the function.
-	if (block.terminator == SPIRBlock::Kill ||
-	    block.terminator == SPIRBlock::TerminateRay ||
-	    block.terminator == SPIRBlock::IgnoreIntersection ||
-	    block.terminator == SPIRBlock::EmitMeshTasks)
+	if (block.terminator == SPIRBlock::Kill || block.terminator == SPIRBlock::TerminateRay ||
```

Comment:
Don't rewrite formatting for no reason here.

Reply:
That goes for the rest of the PR.

Reply:
Done.